questions.md

```diff
@@ -0,0 +1,3 @@
+* What is a pull request?
```

Comment:
👋 Hi @unissechua !

A Pull Request is a way to collaborate with other team members in a repository. It essentially tells your collaborators that you have made some changes to the code and you would like a review.